pipeline-model-extensions/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/withscript/WithScriptScript.java

```diff
@@ -24,12 +24,15 @@
 
 package org.jenkinsci.plugins.pipeline.modeldefinition.withscript;
 
+import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted;
 import org.jenkinsci.plugins.workflow.cps.CpsScript;
 
 import java.io.Serializable;
 
 public abstract class WithScriptScript<T extends WithScriptDescribable<T>> implements Serializable {
+    @Whitelisted
```

Comment:
Once I got past the `PluginFirstClassLoader` problem, I discovered that `WithScriptScript.script` and `WithScriptScript.describable` weren't accessible by https://github.com/jenkinsci/kubernetes-plugin/blob/master/src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/KubernetesDeclarativeAgentScript.groovy, so hey, whitelisting.